pkgs/development/python-modules/lox/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+
+  pathos,
+  pip,
+  pytestCheckHook,
+  pytest-benchmark,
+  pytest-mock,
+  pytest-runner,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "lox";
+  version = "0.12.0";
```

Comment:
```suggestion
  version = "0.12.0";
  pyproject = true;
```

Reply:
Lox is not a pyproject formatted package. This causes the build to fail.

Reply:
you need to provide both `pyproject` and `build-system` for it to work
setting `pyproject` just disable the legacy build, there no need for a `pyproject.toml` to be present (itmight be a bit confusing ik)

Reply:
Do you mean it disables the legacy Python build or the legacy Nix build process? Because the package uses the legacy Python build tools. If we're supposed to call it a pyproject even when it is not, you're correct that that is confusing!

Reply:
Someone else might help giving a better, more precise explaination @natsukagami

Reply:
Resolved this. It also caused tests to run that showed sphinx-rtd-theme is required runtime dep, apparently.